orbital-mechanics/Lecture12/Lecture12.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<p>Second level title for the type of orbit; also write title as "Molniya Orbit".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>